fork/json-schema-form-core/README.md

```diff
@@ -40,21 +44,24 @@ Please contact @Anthropic via our [Gitter](https://gitter.im/json-schema-form/an
 ## Testing it
 
 ### With Angular Schema Form
+
 There is a branch in angular-schema-form called `feature/webpack-babel` that integrates the core.
 To use it roughly follow these steps:
 
-* Clone angular-schema-form to a **sibling** directory and switch to branch `feature/webpack-babel`
-* `npm install` to install dependencies
-* `npm run build` to build with the core.
-* Use dist/angular-schema-form.js, now with the core from this folder. *No need to also load ObjectPath since it is already included*
+- Clone angular-schema-form to a **sibling** directory and switch to branch `feature/webpack-babel`
+- `npm install` to install dependencies
+- `npm run build` to build with the core.
+- Use dist/angular-schema-form.js, now with the core from this folder. _No need to also load ObjectPath since it is already included_
 
 ### With Mocha tests
+
 Tests are written in mocha + chai and run trough `npm test`.
 
 When the command `npm run testdoc` is run instead, the tests will also generate a readable
 `markdown` file [test.md](docs/test.md) to document elements of the library.
 
 ## Notes
-* ObjectPath is bundled with json-schema-form-core
-* angular-schema-form bundles json-schema-form-core so the user doesn't have to include it as an dependency.
-* The code for not using ObjectPath on Angular 1.2 is removed. Could maybe be fixed but I (davidlgj) strongly believe its time to drop Angular 1.2 support since it complicates validation code as well.
+
+- ObjectPath is bundled with json-schema-form-core
+- angular-schema-form bundles json-schema-form-core so the user doesn't have to include it as an dependency.
```

Comment:
```suggestion
- angular-schema-form bundles json-schema-form-core so the user doesn't have to include it as a dependency.
```

Reply:
I did do anything on this file but thanks :D